README.md

```diff
@@ -65,7 +72,7 @@ have it assigned to yourself.
 
 ## Contributing
 
-Check out our [contributing guidelines](docs/CONTRIBUTING.md).
+Check out our [contributing guidelines](https://kinvolk.io/docs/lokomotive/0.5/contributing).
```

Comment:
Maybe we can have some global contributing page then?

Reply:
We have a global cointributon guide, but each project may have its own pages like this. It's okay.